doc/user/content/sql/system-catalog.md

```diff
@@ -249,6 +249,19 @@ Field          | Type       | Meaning
 `schema_id`    | [`bigint`] | The ID of the schema to which the sink belongs.
 `name`         | [`text`]   | The name of the sink.
 
+### `mz_source_info`
+
+The `mz_source_info` table contains a row for each source in the system.
+
+Field          | Type       | Meaning
+---------------|------------|----------
+`source_name`  | [`text`]   | Materialize's internal name for the source.
+`source_id`    | [`text`]   | Materialize's unique ID for the source. Equivalent to `mz_sources.id`.
+`dataflow_id`  | [`bigint`] | The ID of the dataflow responsible for processing this source / partition.
```

Comment:
I'm 99% sure that sources and dataflows are 1:1, but then have multiple partitions. So I think this should be worded as "The ID of the dataflow responsible for processing this source."

Reply:
Sounds good. Looking at sample output appears to show this 1:1 mapping:

```
materialize=> select * from mz_source_info;
  source_name   | source_id | dataflow_id | partition_id | offset |   timestamp   
----------------+-----------+-------------+--------------+--------+---------------
 kafka-u1295/44 | u1295     |          44 | 0            |      5 | 1616627104001
 kafka-u1297/45 | u1297     |          45 | 0            |      5 | 1616627104001
 kafka-u1299/46 | u1299     |          46 | 0            |      5 | 1616627107001
 kafka-u1301/47 | u1301     |          47 | 0            |      2 | 1616627110001
 kafka-u1303/48 | u1303     |          48 | 1            |      1 | 1616627113001
 kafka-u1305/49 | u1305     |          49 | 0            |      1 | 1616627115001
 kafka-u1305/49 | u1305     |          49 | 1            |      1 | 1616627115001
 kafka-u1307/50 | u1307     |          50 | 0            |      1 | 1616627118001
 kafka-u1309/51 | u1309     |          51 | 1            |      1 | 1616627121001
 kafka-u1311/52 | u1311     |          52 | 1            |      2 | 1616627154001
 kafka-u1311/52 | u1311     |          52 | 2            |      1 | 1616627184001
 kafka-u1313/53 | u1313     |          53 | 0            |      1 | 1616627126001
 kafka-u1313/53 | u1313     |          53 | 1            |      2 | 1616627156001
 kafka-u1313/53 | u1313     |          53 | 2            |      1 | 1616627186001
 kafka-u1315/55 | u1315     |          55 | 1            |      2 | 1616627160001
 kafka-u1315/55 | u1315     |          55 | 2            |      1 | 1616627190001
 kafka-u1317/57 | u1317     |          57 | 0            |      7 | 1616627194001
 kafka-u1319/58 | u1319     |          58 | 0            |      7 | 1616627196001
 kafka-u1321/59 | u1321     |          59 | 0            |      8 | 1616627200001
 kafka-u1323/60 | u1323     |          60 | 0            |      8 | 1616627204001
 kafka-u1325/61 | u1325     |          61 | 0            |      8 | 1616627207001
 kafka-u1327/62 | u1327     |          62 | 0            |      1 | 1616627210001
 kafka-u1329/63 | u1329     |          63 | 0            |      4 | 1616627213001
 kafka-u1331/64 | u1331     |          64 | 0            |      3 | 1616627216001
```

Will fix the comment!